src/wp-includes/kses.php

```diff
@@ -1165,15 +1172,47 @@ function wp_kses_attr( $element, $attr, $allowed_html, $allowed_protocols ) {
 	// Split it.
 	$attrarr = wp_kses_hair( $attr, $allowed_protocols );
 
+	// Check if there are attributes that are required.
+	$required_attrs = array_filter(
+		$allowed_html[ $element_low ],
+		function( $required_attr_limits ) {
+			return ! empty( $required_attr_limits['required'] ) && true === $required_attr_limits['required'];
```

Comment:
```suggestion
			return ! empty( $required_attr_limits['required'] );
```

`! empty() === isset() && truthy` -- otherwise use `isset()` if you mean isset.